app/livechat/server/methods/getOpenInquiries.js

```diff
@@ -0,0 +1,21 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
You don't need to create a server method, you can use `LivechatInquiry` in your `endpoint` as we do here:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/imports/server/rest/departments.js#L57